less/footer.less

```diff
@@ -62,6 +62,10 @@
 .footer-color.@{color} {
   background-color: @@color;
   color: @@color-inverted;
+
+  .footer__body a {
+    color: @@color-inverted;
```

Comment:
@kirsty-hames Could also use 'inherit' here.

Reply:
Good shout @swashbuck - amended in [626f9b4](https://github.com/cgkineo/adapt-kineo-footer/pull/19/commits/626f9b485575e5737d8303cf84b63d3be24bb65b)